### ISSUE TITLE
Bug 🐛: Documentation suggestions during Installation

### ISSUE THREAD

**Issue by fortune-max** [ersilia-os/ersilia#249]
### Describe the bug.

Running `ersilia fetch molecular-weight` fails with error when following instructions [here](https://ersilia.gitbook.io/ersilia-book/quick-start/molecular-weight)

`File "/Users/fortune/code/ersilia/ersilia/utils/conda.py", line 297, in clone`
`raise Exception("{0} source environment does not exist".format(src_env))`
`Exception: eosbase-bentoml-0.11.0-py37 source environment does not exist`

### Describe the steps to reproduce the behavior

1. Perform the installation steps as described [here](https://ersilia.gitbook.io/ersilia-book/quick-start/installation).
2. Make sure no conda environment called `eosbase-bentoml-0.11.0-py37` exists with `conda info --env`. (We are not told to create it in documentation above).
3. Run `ersilia fetch molecular-weight`.

### Expected behavior.

`ersilia fetch` and all `ersilia` cli commands fail. Raises this error making going forward very difficult.

`File "/Users/fortune/code/ersilia/ersilia/utils/conda.py", line 297, in clone`
`raise Exception("{0} source environment does not exist".format(src_env))`
`Exception: eosbase-bentoml-0.11.0-py37 source environment does not exist`

(one of the lines above is from a print statement I added to aid me in debugging but is pertinent to the bug).

### Operating environment

MacBook Air M1 2020. macOS Big Sur 11.5.2.

### Additional context

Additional documentation suggestion:
It is unclear if isaura is cloned within the ersilia working dir or in it's parent. Please clarify in [docs](https://ersilia.gitbook.io/ersilia-book/quick-start/installation).

**Comment by anamika-yadav99** [comment]
@fortune-max this has also happened with me once before. I had to pip install bentoml on my own. But that was just once. I have setup environment for ersilia so many times. It never happened again.

**Comment by GemmaTuron** [comment]
Hi @fortune-max 

It looks like a problem with installation on your side. Which Anaconda do you have? and have you created a conda environment with python 3.7, and used the command `pip install -e .`

**Comment by fortune-max** [comment]
Hi Gemma,

The python I had in my conda environment was python3.8 (python3.7 was unavailable for ARM devices and is not on the conda repo). My conda installation is miniconda3 due to size constraints.

I did activate the Ersilia conda environment and run `pip install -e .`.

Hope this helps.

**Comment by GemmaTuron** [comment]
Hi @fortune-max 

This looks like a specific issue of your miniconda installation. I cannot reproduce the error

**Comment by fortune-max** [comment]
Hi @GemmaTuron 

I have duplicated the error again, I added the terminal output for you to see. Hopefully that helps some.
h5py was installed using `conda install h5py` which is why it's already present in pip packages and I removed it from requirements.txt. Other than that the codebase is the same. The conda versions and python versions were also added.

[Ersilia error terminal output.txt](https://github.com/ersilia-os/ersilia/files/8571994/Ersilia.error.terminal.output.txt)

```
(ersilia) fortune@Fortunes-MacBook-Air ersilia % python -V
Python 3.10.4
(ersilia) fortune@Fortunes-MacBook-Air ersilia % conda -V
conda 4.10.1
```
